ravenframework/Optimizers/acquisitionFunctions/LowerConfidenceBound.py

```diff
@@ -0,0 +1,228 @@
+      Should be overwritten by specific acquisition functions
+      @ In, var, np.array, input to evaluate Expected Improvement gradient at
+      @ In, bayesianOptimizer, instance of the BayesianOptimizer cls, provides access to model and evaluation method
+      @ Out, LCBGrad, float/array, LCB gradient value
+    """
+    # NOTE assumes scikitlearn GPR currently
+    meanGrad, stdGrad = bayesianOptimizer._model.supervisedContainer[0].evaluateGradients(var)
+
+    # Retrieve iteration and set epsilon for threshhold
+    self._transient(bayesianOptimizer._iteration[0])
+    beta = ndtri(self._pi)
+
+    # Gradient of LCB
+    LCBGrad = -1*np.transpose(meanGrad) + beta*stdGrad
+
+    return LCBGrad
+
+  def hessian(self, var, bayesianOptimizer):
+    """
+      Evaluates acquisition function's hessian using the current BO instance/ROM
+      Should be overwritten by specific acquisition functions
+      @ In, var, np.array, input to evaluate Expected Improvement hessian at
+      @ In, bayesianOptimizer, instance of the BayesianOptimizer cls, provides access to model and evaluation method
+      @ Out, LCBHess, float/array, LCB hessian value
+    """
+    bayesianOptimizer.raiseAnError(NotImplementedError,'Hessian for Lower Confidence Bound not yet developed')
+
+  #####################
+  # Transient Methods #
+  #####################
+  def exploit(self, iter):
+    """
+      Defines the transient method for exploit setting
+      @ In, iter, current iteration number
+      @ Out, None
+    """
+    self._pi = self._basePi * np.exp(-iter / self._rho)
+
+  def explore(self, iter):
+    """
+      Defines the transient method for explore setting
+      @ In, iter, current iteration number
+      @ Out, None
+    """
+    self._pi = self._basePi * (1 - np.exp(-iter / self._rho))
+
+  def oscillate(self, iter):
+    """
+      Defines the transient method for oscillate setting
+      @ In, iter, current iteration number
+      @ Out, None
+    """
+    self._pi = self._basePi * (np.sin((2 * np.pi * iter) / self._rho)**2)
+
+  def decayingOscillate(self, iter):
+    """
+      Defines the transient method for oscillate setting
+      @ In, iter, current iteration number
+      @ Out, None
+    """
+    self._pi = self._basePi * (np.exp(-iter / (4 * self._rho))) * (np.sin((2 * np.pi * iter) / self._rho)**2)
+
+  # convergence
+  def _converged(self, bayesianOptimizer):
+    """
+      Lower Confidence bound has a different style of convergence
+      @ In, bayesianOptimizer, instance of BayesianOptimizer class
+      @ Out, converged, bool, has the optimizer converged on acquisition?
+    """
+    if self._optValue is None:
+      converged = False
+      return converged
+    optDiff = np.absolute(-1*self._optValue - bayesianOptimizer._optPointHistory[0][-1][0][bayesianOptimizer._objectiveVar])
+    optDiff /= np.absolute(bayesianOptimizer._optPointHistory[0][-1][0][bayesianOptimizer._objectiveVar])
```

Comment:
do you need to check divide by '0' here?

Reply:
If the optimal function value is 0, then yes.